app/api/ddi-index-api/users.js

```diff
@@ -19,21 +21,62 @@ const usersEndpoint = 'users'
  */
 
 /**
+ * @typedef UserAccount
+ * @property {number} id
+ * @property {string} username
+ * @property {number} policeForceId
+ * @property {string} policeForce
+ * @property {boolean|Date} accepted
+ * @property {boolean|Date} activated
+ * @property {boolean|Date} lastLogin
+ * @property {boolean|Date} createdAt
+ */
+
+/**
+ * @typedef GetUserOptions
+ * @property {{ policeForceId?: number }} [filter]
+ * @property {{
+ *    username?: 'ASC'|'DESC';
+ *    policeForce?: 'ASC'|'DESC';
+ *    indexAccess?: boolean;
+ * }} [sort]
+ */
+
+/**
+ * @param {GetUserOptions} options
  * @param callingUser
- * @return {Promise<{ users: UserAccount[], count: number }>}
+ * @return {Promise<{ users: UserAccount[]; count: number }>}
  */
-const getUsers = async (callingUser) => {
+const getUsers = async (options, callingUser) => {
+  const url = new URL(`${usersEndpoint}`, config.ddiIndexApi.baseUrl)
```

Comment:
I think I prefer the baseUrl being added in the base.js rather than here as could be confusing as to whether you need to add it when creating a URL. The nice thing about the base.js was you never had to worry about baseURL, just the path